projects/storefrontapp/src/qualtricsIntegration.js

```diff
@@ -0,0 +1,76 @@
+(function() {
```

Comment:
Do we know where customers will get this JS from?

Reply:
yup, they will get it from their qualtrics web app, and its documented on how to get it in their website, which I will point them to